docs/src/drivers/gm.adoc

```diff
@@ -740,24 +717,24 @@ Every RS485 module pin and parameter name begins as follows:
 gm.<nr of card>.rs485.<module ID>
 ----
 
-where <module ID> is from 00 to 15.
+where _<module ID>_ is from 00 to 15.
 
 .Digital input output module pins
-[width="80%",options="header",cols="<3,^2,<6"]
+[width="100%",options="header",cols="<3,^2,<6"]
 |===
-| Pins                | Type and direction | Pin description
-| .in-<0-7>           | (bit, Out)         | Input pin
-| .in-not-<0-7>       | (bit, Out)         | Negated input pin
+| Pins             | Type and direction | Pin description
+m| .in-_<0-7>_     | (bit, Out)         | Input pin
+m| .in-not-_<0-7>_ | (bit, Out)         | Negated input pin
 |===
 
 .HAL example
-
-----
-gm.0.rs485.0.in-0 # First input of the node.
-# gm.0     - Means the first GM6-PCI motion control card (PCI card address = 0)
-# .rs485.0 - Select node with address 0 on the RS485 bus
-# .in-0    - Select the first digital input module
-----
+|===
+m|gm.0.rs485.0.in-0 |First input of the node.
+2+|constructed from
```

Comment:
I would exclude this from the table, this looks more like a formatting error  :innocent: 
Also ff.